.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/articles/about-codeowners/
```

Comment:
Can we have additional comments? Saw some of the repo such as [aws-xray-daemon](https://github.com/aws/aws-xray-daemon/blob/master/.github/CODEOWNERS), [opentelemetry-collector-contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS), [opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector/blob/main/.github/CODEOWNERS) and here is the general template 
```
#####################################################
#
# List of approvers for this repository
#
#####################################################
#
# Learn about CODEOWNERS file format:
#  https://help.github.com/en/articles/about-code-owners
#
```